dashboard/src/shared/App.test.ts

```diff
@@ -57,4 +57,20 @@ describe("App", () => {
       expect(await App.listApps("default", true)).toEqual(apps);
     });
   });
+  describe("delete", () => {
+    it("should delete an app in a namespace", async () => {
+      moxios.stubRequest("/api/tiller-deploy/v1/namespaces/default/releases/foo", {
+        response: "ok",
+        status: 200,
+      });
+      expect(await App.delete("foo", "default", false)).toBe("ok");
+    });
+    it("should delete and purge an app in a namespace", async () => {
+      moxios.stubRequest("/api/tiller-deploy/v1/namespaces/default/releases/foo?purge=true", {
+        response: "ok",
+        status: 200,
+      });
+      expect(await App.delete("foo", "default", true)).toBe("ok");
+    });
+  });
```

Comment:
just noticed we don't have any tests where responses are not 200, should we add those?